app/feed-event-service/feed-event.mocks.ts

```diff
@@ -0,0 +1,23 @@
+import {FeedEvent} from "../feed-event/feed-event";
+import {DOSE_AMIGOS_USERS} from "../dose-amigos-user-service/dose-amigos-user-mocks";
+
+export var FEED_EVENTS: Array<FeedEvent> = [
+    {
+        id: 3,
+        actionDateTime: DOSE_AMIGOS_USERS[0].lastTimeDoseTaken,
+        user: DOSE_AMIGOS_USERS[0],
+        action: "has completed their medication."
+    },
+    {
+        id: 2,
+        actionDateTime: DOSE_AMIGOS_USERS[1].lastTimeDoseTaken,
+        user: DOSE_AMIGOS_USERS[1],
+        action: "has completed their medication."
+    },
+    {
+        id: 1,
+        actionDateTime: DOSE_AMIGOS_USERS[2].lastTimeDoseTaken,
+        user: DOSE_AMIGOS_USERS[2],
+        action: "has completed their medication."
```

Comment:
On the wiki: https://github.com/dose-amigos/dose-amigos/wiki/Data-Objects-we'll-need#doseevent
`DoseEvent` says `action (Enum: TAKEN, SKIPPED, MISSED)`
`FeedEvent` says `action (String)`

They are the same?

What are some of the messages we expect to see in the News Feed on cards? I am trying to ensure I have all the data necessary.